tools/ci_build/github/apple/build_ios_framework.py

```diff
@@ -15,11 +15,11 @@
 REPO_DIR = os.path.normpath(os.path.join(SCRIPT_DIR, "..", "..", "..", ".."))
 BUILD_PY = os.path.join(REPO_DIR, "tools", "ci_build", "build.py")
 
-# We by default will build below 2 archs
-DEFAULT_BUILD_OSX_ARCHS = [
-    {'sysroot': 'iphoneos', 'arch': 'arm64'},
-    {'sysroot': 'iphonesimulator', 'arch': 'x86_64'},
-]
+# We by default will build below 3 archs
```

Comment:
>3  [](http://example.com/codeflow?start=33&length=2)

nit: if we don't mention the number it won't need to be updated again. could make this change next time though

Reply:
Will remove in a later PR